Makefile

```diff
@@ -86,29 +86,29 @@ test-unit: ## Run the unit tests
 	$(Q)go test -count=1 -short $(UNIT_TEST_DIRS)
 
 test-e2e: ## Run the e2e tests
-	$(Q)go test -count=1 ./test/e2e
+	ginkgo test ./test/e2e
 
 ###################
 # Running Targets #
 ###################
 .PHONY: deploy teardown run run-local run-e2e run-e2e-local
+
+IMAGE_LOAD_COMMAND=kind load docker-image
+load-image: ## Load-image loads the currently constructed image onto the cluster
+	$(IMAGE_LOAD_COMMAND) $(IMAGE)
+
 deploy: generate ## Deploy the Combo operator to the current cluster
 	kubectl apply --recursive -f manifests
 
+deploy-local: generate ## Deploy the Combo operator to the current cluster
+	kubectl apply --recursive -f manifests
```

Comment:
What's the difference between the deploy and deploy-local targets?

Reply:
Looks like some left over debugging statement, remove it now.